2025-03-04 20:25:58,090 [INFO] 程序已以管理员权限运行。
2025-03-04 20:25:58,091 [INFO] step 3: delay power off
2025-03-04 20:25:58,091 [INFO] step 4: bitlocker unlock
2025-03-04 20:26:09,971 [ERROR] 详细错误信息：
Traceback (most recent call last):
  File "t_Tcg_BitLocker_10_UnlockAPL.py", line 120, in Run
    current_locked_state = (
KeyError: '锁定状态'
